src/components/GameView/GameOverDialog.vue

```diff
@@ -90,6 +124,27 @@ export default {
 
       return 'loss-img';
     },
+    currentMatch(){
+      return this.game.currentMatch
+    },
+    matchGameStats(){
+      if(!this.game.currentMatch?.games) return []
+      const result= this.game.currentMatch.games.map(game=>{
+        if (game.result === -1) return 'I'
+        if(game.result === 0 || game.result === 1){
+          if(game.p0 === this.player.id) return 'W'
+          return 'L'
+        }
+        return 'D'
+      })
+      return result
+    },
+    currentMatchWinner() {
+      return this.game.currentMatch?.winner
+    },
+    playerWinsMatch(){
+      return this.game.currentMatch?.winner === this.player.id
```

Comment:
Make sure to always include a semicolons to end a statement. @itsalaidbacklife we should do a pass on our linting rules and get some precommit linting in place to keep a consistent coding style as a follow up.

Reply:
https://github.com/cuttle-cards/cuttle/issues/340

Reply:
I have some thought around this to lint only the staged files since it doesn’t take long at all but I’ll proof it out first to make sure we don’t cause too much of a disruption. We definitely need it for consistency though.